docs/changelog.rst

```diff
@@ -14,6 +14,12 @@ New Features
 ~~~~~~~~~~~~
 - Added support for :attr:`fortnite_api.CosmeticType.SHOES`.
 
+Miscellaneous
+~~~~~~~~~~~~~
+- Add safeguards against Fortnite changes or messed up values within the API.
```

Comment:
I'd also note that all "Enum-like" classes in the breaking changes section are not actually `typing.Enums` anymore, in case one of our users is using an instance check.

Reply:
Shouldn't instance checks still work, since we just subclassing Enum? But yeah, if you exspect a specific type it could break. Gonna add that to the changelog.

Reply:
I updated the changelog. Feel free to suggest changes in wording.